repairnator/repairnator-dockerpool/src/main/java/fr/inria/spirals/repairnator/dockerpool/Launcher.java

```diff
@@ -198,7 +199,9 @@ private void runPool() {
         ExecutorService executorService = Executors.newFixedThreadPool(this.arguments.getInt("threads"));
 
         for (Integer builId : buildIds) {
-            RunnablePipelineContainer runnablePipelineContainer = new RunnablePipelineContainer(imageId, builId, logFile, this.arguments.getString("runId"));
+            //@fermadeiral: I put "./client_secret.json" as a temporary thing. I will fix that when your changes be merged.
+            GoogleSpreadSheetTreatedBuildTracking googleSpreadSheetTreatedBuildTracking = new GoogleSpreadSheetTreatedBuildTracking(builId, "./client_secret.json");
```

Comment:
You should not do that: add a new argument in Dockerpool Launcher, and change the repairnator_launcher.sh accordingly. Have a look on the other PR if you want. This one will certainly be merged before the other one.